app/src/main/java/com/taf/shuvayatra/ui/activity/DestinationDetailActivity.java

```diff
@@ -64,6 +69,15 @@ public void onCreate(Bundle savedInstanceState) {
         if (bundle != null) {
             Logger.e(TAG, ": " + bundle.containsKey(MyConstants.Extras.KEY_COUNTRY));
             mCountry = (Country) bundle.get(MyConstants.Extras.KEY_COUNTRY);
+            if (mCountry == null) {
+                mCountry = new Country();
+                mCountry.setId(bundle.getLong(MyConstants.Extras.KEY_COUNTRY_ID));
+                mCountry.setTitle(bundle.getString(MyConstants.Extras.KEY_COUNTRY_TITLE));
```

Comment:
@ngima we could have just sent the id instead of these extra parameters? and compared ids with the country list provided below